lib/msal-browser/src/request/PopupRequest.ts

```diff
@@ -48,4 +49,5 @@ export type PopupRequest = Partial<
     scopes: Array<string>;
     popupWindowAttributes?: PopupWindowAttributes;
     popupWindowParent?: Window;
+    overrideInteractionInProgress?: boolean;
```

Comment:
Please add documentation for this with recommended usage pattern and appropriate warnings about the tradeoffs. This is a footgun and folks should be super careful about how and when they use it.

Reply:
Added documentation and migration guide